src/validation/custom.validation.ts

```diff
@@ -97,6 +97,14 @@ export const checkSecondNationality = (nationality: string = "", secondNationali
   return true;
 };
 
+export const checkPublicRegisterJurisdiction = (public_register_name: string = "", public_register_jurisdiction: string = "") => {
+  const separator = ",";
+  if (public_register_name && public_register_jurisdiction && `${public_register_name}${separator}${public_register_jurisdiction}`.length > 160) {
```

Comment:
I can't see any new testing around this.

I'd expect tests when total length is exceeded and not exceeded. When one of the fields is over 158 characters (presume other must be at least 1 + comma) and another when both are over 160, plus any other scenarios you can think of.

Reply:
Added tests to focus on just this.